.github/workflows/node.js.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build_and_test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 15.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: yarn
```

Comment:
Of the 1:27 this takes, installing the dependencies takes a full minute. I wonder if there's a way to cache them or something. :thinking: 

To be clear, I'm definitely considering it out of scope for this PR. But if it's easy, that'd be awesome. Because you know we won't get around to making that improvement later. :sweat_smile:

Reply:
Caching is now implemented! We went from 1m31s without the cache down to 35s once the cache had been created :tada:

Reply:
Nice! Was that the `uses: actions/cache@v2`?